.vscode/launch.json

```diff
@@ -0,0 +1,23 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Attach to Backend",
+            "type": "debugpy",
+            "request": "attach",
```

Comment:
Can we use the `justMyCode` attribute here? I usually set this to `false` so I can trace the path to other dependencies as well. For example, one of the dependencies here is `lumigator_schemas`, which is also *our* code.